### PR TITLE
Update 2020-02-07-the-ember-times-issue-134.md

### PR DESCRIPTION
**Pull request by MinThaMie**
I'm a bit unsure about my intro text, so please let me know if you know something better :)

### CODE REVIEW

File: source/2020-02-07-the-ember-times-issue-134.md [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
-<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+How structuring your Figma can make your developer life easier 🎨,
 
 READMORE
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [From Figma to Ember to Storybook … with 2 Clicks 🎨](https://gos.si/blog/from-figma-to-ember-to-storybook-with-2-clicks/)
+[Thomas Gossmann @gossi](https://github.com/gossi) has started a blog post series on Figma, Ember and Storybook. It is a 3 part series. [Part 1](https://gos.si/blog/full-featured-themes-in-figma/) has just been publish and will tell you all about how to organise your design in Figma. So if you want to learn something about themes and design tokens, go ahead and read  [Part 1: Full Featured Themes in Figma](https://gos.si/blog/full-featured-themes-in-figma/) .
```
Comment (jayjayjpg) [inline source/2020-02-07-the-ember-times-issue-134.md]: To make both the markdown linter happy and to round up the sentence, I think it'd be:

```suggestion

[Thomas Gossmann @gossi](https://github.com/gossi) has started a blog post series on Figma, Ember and Storybook. It is a 3 part series. [Part 1](https://gos.si/blog/full-featured-themes-in-figma/) has just been published and will tell you all about how to organise your design in Figma. So if you want to learn something about themes and design tokens, go ahead and read  [Part 1: Full Featured Themes in Figma](https://gos.si/blog/full-featured-themes-in-figma/) .
```

What do you think?

**Review by jayjayjpg — APPROVED** [review]
This reads great! I left one more small suggestion, otherwise this is already good to go! Thank you for another :sparkles: writeup this week!

File: source/2020-02-07-the-ember-times-issue-134.md [hunk 2]
```diff
@@ -1,26 +1,25 @@
...
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+Part 2 on design tokens in Ember and part 3 on documentation within the Ember Ecosystem are on their way. Follow [@unistyler](https://twitter.com/unistyler) on Twitter to keep posted on when they are published.
```
Comment (ijlee2) [inline source/2020-02-07-the-ember-times-issue-134.md]: I think "be notified" sounds better here. You'd keep someone posted when you are the person who makes an announcement. :)
```suggestion
Part 2 on design tokens in Ember and part 3 on documentation within the Ember Ecosystem are on their way. Follow [@unistyler](https://twitter.com/unistyler) on Twitter to be notified when they are published. 
```

**Review by ijlee2 — APPROVED** [review]
Looks great! In addition to Jessica's feedback, I left one more. Thanks for an early start!

File: source/2020-02-07-the-ember-times-issue-134.md [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
 READMORE
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [From Figma to Ember to Storybook … with 2 Clicks 🎨](https://gos.si/blog/from-figma-to-ember-to-storybook-with-2-clicks/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+[Thomas Gossmann @gossi](https://github.com/gossi) has started a blog post series on Figma, Ember and Storybook. It is a 3 part series. [Part 1](https://gos.si/blog/full-featured-themes-in-figma/) has just been published and will tell you all about how to organise your design in Figma. So if you want to learn something about themes and design tokens, go ahead and read  [Part 1: Full Featured Themes in Figma](https://gos.si/blog/full-featured-themes-in-figma/) .
```
Comment (ijlee2) [inline source/2020-02-07-the-ember-times-issue-134.md]:
```suggestion
[Thomas Gossmann @gossi](https://github.com/gossi) has started a blog post series on Figma, Ember and Storybook. It is a 3 part series. [Part 1](https://gos.si/blog/full-featured-themes-in-figma/) has been published and will tell you all about how to organise your design in Figma. So if you want to learn something about themes and design tokens, go ahead and read  [Part 1: Full Featured Themes in Figma](https://gos.si/blog/full-featured-themes-in-figma/) .
```

**Pull request merged**